fix cursor ends up in wrong location #2353

kpaharelau opened the pull request:
It is very strange but Robot mono font doesn't work correctly with codemirror editor. I have changed robot mono on ubuntu mono font.

kpaharelau committed:
fix cursor ends up in wrong location #2353

It is very strange but Robot mono font doesn't work correctly with codemirror editor. I have changed robot mono on ubuntu mono font.

scottdraves commented on core/src/main/web/app/styles/vendor_overrides.scss:
```diff
@@ -26,6 +26,7 @@
 
 .CodeMirror pre {
   z-index: initial;
+  font-size: 14px;
```
why is this needed?

kpaharelau committed:
fix cursor ends up in wrong location #2353

OK this looks good. Please remove the Roboto Mono font, and everywhere it appeared replace it with Ubuntu Mono. - fixed
